admin/package.json

```diff
@@ -70,5 +70,8 @@
     "webpack-cli": "^3.2.3",
     "webpack-dev-server": "^3.1.14",
     "worker-loader": "^2.0.0"
+  },
+  "overrides": {
+    "three": "github:mozillareality/three.js#hubs/master"
```

Comment:
I think this should match what is in the main hubs package.json

Reply:
Makes sense. I think I just happen to open this PR when we were in the middle of a three.js update.